onlinejudge/service/yukicoder.py

```diff
@@ -256,6 +256,18 @@ def download_sample_cases(self, *, session: Optional[requests.Session] = None) -
                 samples.add(data.encode(), name)
         return samples.get()
 
+    def get_handmade_sample_cases(self, *, html: str) -> List[TestCase]:
```

Comment:
Since this function is useful for other tests, it should be placed in `test/utils.py`.